tools/accuracy_checker/configs/face-detection-0105.yml

```diff
@@ -0,0 +1,39 @@
+models:
+  - name: face-detection-0105
+
+    launchers:
+      - framework: dlsdk
+        tags:
+          - FP32
+        model:   intel/face-detection-0105/FP32/face-detection-0105.xml
```

Comment:
please add also FP16 section for all new AC configs

Reply:
added